cmd/build.go

```diff
@@ -147,17 +146,25 @@ func (cmd *BuildCmd) build() (v1.Image, error) {
 		RunV2:             true,
 		NoPush:            true,
 		KanikoDir:         "/.dockerless",
+		Cache:             true,
 		CacheRunLayers:    true,
 		CacheCopyLayers:   true,
 		CompressedCaching: true,
 		SkipUnusedStages:  true,
 		Compression:       config.ZStd,
 		CompressionLevel:  3,
 		CacheOptions: config.CacheOptions{
-			CacheDir: "/.dockerless/cache",
 			CacheTTL: time.Hour * 24 * 7,
 		},
-	})
+	}
+	if cmd.Registry != "" {
+		opts.CacheRepo = cmd.Registry
```

Comment:
tiny: what do you think about keeping vars consistent and just naming it `cmd.CacheRegistry`?

Reply:
yes 100%